listenbrainz_spark/stats/user/listening_activity.py

```diff
@@ -124,7 +155,15 @@ def calculate_listening_activity():
 
 
 def get_listening_activity(stats_range: str):
-    """ Calculate the number of listens of users for the given stats_range """
+    """ Calculate the number of listens of users for the given stats_range.
+
+    .. note::
+        Compared to other statistics, this stat needs listens for twice the time period.
+        For eg: for week range, this statistic needs the listen data of the last 2 weeks.
+        It will calculate individual listens for each weekday and send to LB webserver.
+        The LB frontend presents a chart comparing the listening activity on corresponding
+        weekdays of each week.
+    """
     logger.debug(f"Calculating listening_activity_{stats_range}")
```

Comment:
Yes, that should be possible. The same should be done for other types of stats. However, this will need rewriting many stats tests so I'd prefer to take that up separately. I'd suggest to create a ticket for now and do this later. Thoughts?

Reply:
sure, happy to create a ticket for this